test/integration/i18n-support/test/shared.js

```diff
@@ -35,6 +35,27 @@ async function addDefaultLocaleCookie(browser) {
 }
 
 export function runTests(ctx) {
+  if (ctx.basePath) {
+    it.only('should handle basePath like pathname', async () => {
```

Comment:
Let's use `it()` instead so it doesn't skip all the test

Reply:
Updated in https://github.com/vercel/next.js/pull/31456